src/components/__tests__/LibraryDetailPage-test.tsx

```diff
@@ -108,6 +98,17 @@ describe("LibraryDetailPage", () => {
     expect(infoLabels.indexOf("description")).to.equal(-1);
   });
 
+  it("should not display empty tabs", () => {
+    let tabs = wrapper.find(".tab-nav");
+    expect(tabs.length).to.equal(3);
+    expect(wrapper.find(".tab-navs").text()).to.contain("Areas");
+    let libraryWithoutAreas = {...library, ...{areas: {focus: [], service: []}}};
```

Comment:
I think here you can also remove the spread for `areas`.

Reply:
You want to use it for objects where you want all the attributes but not necessary for the attribute you want to override.